app/brave_settings_strings.grdp

```diff
@@ -1550,6 +1550,12 @@
   <message name="IDS_SETTINGS_LEO_ASSISTANT_SHOW_IN_CONTEXT_MENU_DESC" desc="The description for settings option to show Leo in context menu">
     Leo will appear in the context menu when you right-click on a website. This will send context from the current web page to Leo.
   </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_AUTOMATICALLY_ATTACH_PAGE_LABEL" desc="The text for settings option to automatically attach page to Leo">
```

Comment:
These two strings are for the settings toggle here:
![image](https://github.com/user-attachments/assets/ff77ff75-0edd-4d7a-81e7-39be2879b014)

Reply:
we should decide on tab vs page (see comment below) and be consistent in copy throughout leo